source/isaaclab_tasks/test/test_environments.py

```diff
@@ -41,6 +41,14 @@ def setUpClass(cls):
                 cls.registered_tasks.append(task_spec.id)
         # sort environments by name
         cls.registered_tasks.sort()
+        # TODO: fix these tests!!
```

Comment:
What is broken around these environments? The stack ones come from mimic team right?

Reply:
Yeah would be against merging this. We should fix the environments asap instead.